### DISCUSSION TITLE
Crazyflie python library examples no longer work after installing custom firmware

### DISCUSSION THREAD

**Discussion by Fygas1** [bitcraze/discussions#1620] [Q&A] [answered]
Hello,

I've been experimenting with Crazyflie and I installed the push demo custom firmware found here https://github.com/bitcraze/crazyflie-firmware/tree/master/examples/demos/app_push_demo

Everything works as intended except running Python lib examples using the Crazyradio. For example this one: https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/positioning/flowsequenceSync.py

When running this, the light on the Crazyflie that indicates that its now connected with the Crazyradio lights on but nothing happens. I tried installing the official firmware using the Client so that the custom firmware is removed and then tried to run this python example and it worked. I then tried again to install the push demo custom firmware and now again, the python example doesn't work.

Is this expected behavior when installing custom firmware? If so, is there any way for both custom firmware and python library that communicates with the Crazyflie work together?

Thanks in advance

**Comment by gemenerik** [comment, marked as answer]
Hi Andreas,

That's a good question! Examples (whether they are implemented as an app, or in the cflib) are not designed to run simultaneously, so it's possible that they are incompatible. In fact, both the examples you mentioned will attempt to make the drone fly, which would explain the conflict you're experiencing. Unfortunately, you won't be able to use the custom firmware and the Python library examples together in this case. You could, of course, modify either one to merge the behavior.

Hope that helps!

**Reply by Fygas1** [reply]
Thanks for your answer Rik. Really appreciate it!
